StreamChatArtifacts.json

```diff
@@ -0,0 +1,3 @@
+{
+	"4.2.0": "https://github.com/GetStream/stream-chat-swift/releases/download/2.6.1/StreamChat-All.zip"
```

Comment:
Let's get rid of this "test" release here.

Reply:
Good catch